lib/rubocop/extension/generator/generator.rb

```diff
@@ -138,7 +138,7 @@ def self.defaults!
           RUBY
 
           if Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.3.9')
-            patch 'README.md', /\$ bundle add #{name}$/, "\$ bundle add #{name} --require=false"
+            patch 'README.md', /\$ bundle add (.*)$/, '$ bundle add \1 --require=false'
```

Comment:
Besides the fact it allows us to simply support both versions, another reason not to hard code the value is that rubygems/rubygems#6338 changes it again.